x/nft/blockchain/handler.go

```diff
@@ -47,7 +50,17 @@ func (h IssueHandler) Deliver(ctx weave.Context, store weave.KVStore, tx weave.T
 	if err != nil {
 		return res, err
 	}
-	o, err := h.bucket.Create(store, weave.Address(msg.Owner), msg.Id, msg.Approvals, msg.Details.Networks)
+	//TODO: Need to discuss, maybe we need to also validate the linked blockchainID vs ours
```

Comment:
yes that is a good sanity check, we should do that validation. Otherwise we could link any persistent ticker.